tests/integration/conftest.py

```diff
@@ -40,10 +40,11 @@ def app():
     app = create_app(
         DEBUG=True,
         WTF_CSRF_ENABLED=False,
-        CELERY_ALWAYS_EAGER=True,
+        CELERY_ALWAYS_EAGER=False,
```

Comment:
But this is the problem! We MUST do this, otherwise the Celery application has already been created with the wrong configuration, but we don't want to modify the configuration for all the other tasks. So the solution is to create another application fixture, something which I was trying to avoid (https://github.com/inspirehep/inspire-next/issues/1843).